tests/kernel/common/testcase.yaml

```diff
@@ -2,3 +2,6 @@ tests:
   kernel.common:
     tags: kernel
     min_ram: 16
+  kernel.common.bootdelay:
+    extra_configs:
+      - CONFIG_BOOT_DELAY=10
```

Comment:
This is being turned on.
But how do we know it actually works? Tests need to prove that the feature works.